Firestore/core/src/firebase/firestore/remote/serializer.cc

```diff
@@ -41,7 +46,14 @@ void EncodeUnsignedVarint(pb_ostream_t* stream,
   }
 }
 
-uint64_t DecodeUnsignedVarint(pb_istream_t* stream) {
+/**
+ * Decodes a varint. Note that (despite the return type) this works for bool,
+ * enum, int32, int64, uint32 and uint64 proto field types.
+ *
+ * @return The decoded varint as a uint64_t (though might actually be a bool,
+ * enum int32, int64, uint32 or uint64 proto field type.)
+ */
+uint64_t DecodeVarint(pb_istream_t* stream) {
```

Comment:
All the same comments as its counterpart function.

Reply:
Done.